sqe-http-api/Controllers/ArtefactController.cs

```diff
@@ -24,20 +22,48 @@ public ArtefactController(IUserService userService, IArtefactService artefactSer
             this._userService = userService;
         }
         
+        private void ParseOptionals(List<string> optionals, out bool images, out bool masks)
+        {
+            images = masks = false;
+            if (optionals == null) return;
```

Comment:
This is very petty, but I prefer to put return in a separate line...
Also, I want to see what's the difference between Githubs "Add Single command" and "Start a review"

Reply:
Fine with me.  Changed 6 instances.